testenv/docker/krb5kdc-latest/README.md

```diff
@@ -4,11 +4,11 @@ DO NOT USE THIS CONTAINER FOR ANY PRODUCTION USE!!!
 
 To run:
 ```bash
-docker run -v /etc/localtime:/etc/localtime:ro -p 98:88 -p 98:88/udp --rm --name gokrb5-kdc-latest jcmturner/gokrb5:kdc-latest &
+docker run -v /etc/localtime:/etc/localtime:ro -p 98:88 -p 98:88/udp --rm --name gokrb5-kdc-latest grafana/gokrb5-test:kdc-latest &
 ```
 
 To build:
 ```bash
-docker build -t jcmturner/gokrb5:kdc-latest --force-rm=true --rm=true .
-docker push jcmturner/gokrb5:kdc-latest
+docker build -t grafana/gokrb5-test:kdc-latest --force-rm=true --rm=true .
+docker push grafana/gokrb5-test:kdc-latest
```

Comment:
Shouldn't be `ghcr`?

Reply:
Yeah, good catch! You can see it in #2 - thanks! 👍🏻